backend/app/routers/authentication.py

```diff
@@ -206,6 +206,42 @@ async def revoke_admin(
         )
 
 
+@router.post("/users/enable_readonly/{useremail}", response_model=UserOut)
+async def enable_readonly_user(
+    useremail: str, current_username=Depends(get_current_user), admin=Depends(get_admin)
+):
+    if admin and current_username.admin:
+        if (user := await UserDB.find_one(UserDB.email == useremail)) is not None:
+            user.read_only_user = True
```

Comment:
you might not want to make an admin user "read_only_user" and vice versa?

Reply:
fixed. cannot make an admin user read only